.eslintrc.json

```diff
@@ -103,6 +104,12 @@
         "jsdoc/require-jsdoc": "off",
         "jsdoc/require-param": "off",
         "jsdoc/require-returns": "off",
+        "mocha/handle-done-callback": ["error"],
+        "mocha/no-async-describe": ["error"],
+        "mocha/no-mocha-arrows": ["error"],
+        "mocha/no-return-from-async": ["error"],
+        "mocha/no-sibling-hooks": ["error"],
+        "prefer-const": ["error"],
```

Comment:
This could go in the root `rules` definition; it should apply everywhere IMO. Fixing those issues would likely be another set of changes, though.

Reply:
Specifically, `prefer-const` ? I'll try that in the root rules and fix if it isn't too many changes

Reply:
Yep - sorry, that wasn't clear. `prefer-const` should apply everywhere.